### DISCUSSION TITLE
[Projects Beta] Integration with repository issues

### DISCUSSION THREAD

**Discussion by adamkdean** [community/community#8519] [Projects and Issues]
In repo issues, you're unable to filter by new project beta boards.

Also, it would be great if you could select all issues in a repo issue list and assign them to a project, the same way you can add labels and close them, etc. Either this, or by adding an "add all issues from repo" feature to the beta project board.

**Comment by gordsport** [comment]
Looks like this is a dupe of: https://github.com/github/feedback/discussions/8326 (upvoted both 🚀 )

**Reply by adamkdean** [reply]
Good spot, sorry for the dupe, but on the other hand, shows that others have spotted it too :)
